recipes/watermark/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "watermark" %}
+{% set version = "1.3.1" %}
+{% set md5 = "cf25757d942e3a2ae5077b2ffff172dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - ipython
+    - certifi
+
+  run:
+    - python
+    - ipython
+    - certifi
+
+test:
+  imports:
+    - watermark
+    - certifi
+
+about:
+  home: https://github.com/rasbt/watermark
+  license: newBSD
+  license_file: LICENSE
```

Comment:
Please add a comment explaining there is no license file yet with a link to issue ( https://github.com/rasbt/watermark/issues/20 ) and comment this line for now.